github_server/action.yml

```diff
@@ -0,0 +1,53 @@
+name: "ALKiln/github_server: Isolated docassemble tests"
+description: "Automatically test docassemble interviews on GitHub's servers, in an isolated environment. Avoids docassemble reload errors and other external factors"
+
+
+inputs:
```

Comment:
Do the regular ALKiln `inputs` of the package authors need to be handled in any special way to pass them on to the default ALKiln action?

Reply:
Secrets should be affected, as they're all read from env vars, but we do need to manually pass these args to the `uses: SuffolkLITLab/ALKiln` step below

Reply:
Can confirm. This duplicates the work we do in ALKiln. I wonder if there's another way to do this. Also, I'm concerned about the custom env vars that users are setting - will they come through. I suspect they'll be fine, but we'll be able to test that later. Given that, would it be better to have authors set the ALKiln as env vars instead of passing them as inputs. It would be a breaking change, but it would save a lot of duplication and maintenance cost.

Reply:
Possible argument for keeping inputs: It makes those vars distinct from other vars - they clearly belong to ALKiln and we are able to make some required. Maybe they're also (only a little) easier to look up in the documentation. I'm not as sure about the ones that aren't required.

Reply:
Conclusion: (not about inputs in general) We made a separate workflow script that authors can make to use the server action. In their workflow, the set up the server with one action and then run ALKiln with the other action, with special arguments.